src/test/generate-matrix.py

```diff
@@ -137,7 +137,7 @@ def __str__(self):
 # Ubuntu: coverage
 matrix.add_build(
     name="coverage",
-    env=dict(PYTHON_VERSION="3.7"),
+    env=dict(PYTHON_VERSION="3.6"),
```

Comment:
if `PYTHON_VERSION` is not required by flux-accounting, I wonder if you can just drop this line? Since 3.6 is the default python3, it should be picked up by default by the flux-accounting configure?

Reply:
I can try this now!